spring-cloud-gcp-samples/spring-cloud-gcp-integration-pubsub-sample/spring-cloud-gcp-integration-pubsub-sample-test/pom.xml

```diff
@@ -111,6 +111,15 @@
 					<skip>true</skip>
 				</configuration>
 			</plugin>
+			<plugin>
+				<groupId>org.graalvm.buildtools</groupId>
+				<artifactId>native-maven-plugin</artifactId>
+				<configuration>
+					<buildArgs>
+						<arg>-H:+RunReachabilityHandlersConcurrently</arg>
+					</buildArgs>
+				</configuration>
+			</plugin>
```

Comment:
This can be removed since we are skipping this test. Keep in temporarily to mark this test is also affected by the `java.lang.ClassNotFoundException: kotlinx.serialization.StringFormat` error.